srv/test/resources/zephyr/prj.conf

```diff
@@ -1,3 +1,5 @@
 CONFIG_TEST_ENTRY3=y
 CONFIG_TEST_ENTRY1="ABC"
 CONFIG_HIDDEN_ENTRY=y
+
+CONFIG_BT_MESH_DEBUG=n
```

Comment:
You could also make the test insert this change just for this test case, to avoid having to redo all the other tests, but I'm okay either way:
```py

    notify(
        'textDocument/didChange', {
            'textDocument': {
                'uri': str(Uri.file(path.join(zephyr_root, 'prj.conf'))),
                'version': 1,
            },
            'contentChanges': [{
                'range': {
                    'start': {
                        'line': 5,
                        'character': 0,
                    },
                    'end': {
                        'line': 5,
                        'character': 0,
                    }
                },
                'text': 'CONFIG_BT_MESH_'
            }]
        })
```